dbcon/mysql/ha_mcs_impl.cpp

```diff
@@ -1916,7 +1949,7 @@ uint32_t doUpdateDelete(THD* thd, gp_walk_info& gwi, const std::vector<COND*>& c
     CalpontSystemCatalog::ROPair roPair;
     try
     {
-        roPair = csc->tableRID( aTableName );
+        roPair = csc->tableRID(aTableName);
```

Comment:
This should have `lower_case_table_names` as the second argument.

Reply:
aTableName was converted at line 1580